apps/beeai-ui/src/components/AgentsNav/AgentsNav.tsx

```diff
@@ -13,16 +13,19 @@ import { routes } from '#utils/router.ts';
 
 export function AgentsNav() {
   const pathname = usePathname();
+  const searchParams = useSearchParams();
   const { transitionTo } = useRouteTransition();
 
   const { data: agents } = useListAgents({ onlyUiSupported: true, sort: true });
 
   const items: NavItem[] | undefined = agents?.map(({ name }) => {
     const route = routes.agentRun({ name });
+    const fullPath = `${pathname}?${searchParams.toString()}`;
+
     return {
       key: name,
       label: name,
-      isActive: pathname === route,
+      isActive: fullPath === route,
```

Comment:
1. Would be easier to compare just agent names.
2. What happened to the `useAgentNameFromPath` hook? It could be used to get current agent.